source/content/partials/traffic-overages-faq.md

```diff
@@ -24,7 +24,7 @@ We encourage all customers to take advantage of [Traffic Limits and Overages](/g
 
 ### What about legitimate traffic spikes?
 
-The internet can make any website famous overnight, and this may not be under your control. Pantheon's platform is designed to support such events, and it's one of the main reasons people choose us to run their sites. Luckily, traffic spikes are easily discernible and we take this into account when monitoring overages. Sustained traffic overages will continue to require an adjustment to your pricing plan as set out above.
+The internet can make any website famous overnight, and this may not be under your control. Pantheon's platform is designed to support such events, and it's one of the main reasons people choose us to run their sites. It is because of uncontrollable events likr this that all Performance & Elite plans on an annual agreement recieve at least one month of overage protection (qualified nonprofits recieve four months of protection) per calendar year.  Sustained traffic overages will be enforced and may require a change to your site plan should the traffic levels see a sustained increase.
```

Comment:
```suggestion
The internet can make any website famous overnight, and this may not be under your control. Pantheon's platform is designed to support such events, and it's one of the main reasons people choose us to run their sites. It is because of uncontrollable events like this that all Performance & Elite plans on an annual agreement receive at least one month of overage protection (qualified nonprofits receive four months of protection) per calendar year.  Sustained traffic overages will be billed and may require a change to your site plan should the traffic levels see a sustained increase.
```

Reply:
@IngridKwok  @MystikSpiral-74  @carly-armstrong I think "enforcement" is an unnecessarily vague and somewhat threatening word for what we're doing which is billing accounts for visits that exceed their plan. Can we just say "billed" when we mean "billed"

Reply:
@stevector @carly-armstrong  We can say:
apply site visitor overages
overage will be billed
pay overage fees

Reply:
@MystikSpiral-74 @carly-armstrong @stevector Please confirm we can publish this.